common/test/acceptance/tests/lms/test_problem_types.py

```diff
@@ -535,6 +535,35 @@ def answer_problem(self, correctness):
         else:
             self.problem_page.click_choice("choice_choice_2")
 
+    @attr(shard=7)
+    def test_can_show_answer(self):
+        """
+        Scenario: Verifies that show answer button is working as expected.
+
+        Given that I am on courseware page
+        And I can see a CAPA problem with show answer button
+        When I click "Show Answer" button
+        The correct answer is displayed with a single correctness indicator.
+        """
+        # Click the correct answer, but don't submit yet. No correctness shows.
+        self.answer_problem('correct')
+        self.assertFalse(self.problem_page.is_correct_choice_highlighted(correct_choices=[3]))
+
+        # After submit, the answer should be marked as correct.
+        self.problem_page.click_submit()
+        self.assertTrue(self.problem_page.is_correct_choice_highlighted(correct_choices=[3]))
+
+        # Switch to an incorrect answer. This will hide the correctness indicator.
+        self.answer_problem('incorrect')
+        self.assertFalse(self.problem_page.is_correct_choice_highlighted(correct_choices=[3]))
+
+        # Now click Show Answer. A single correctness indicator should be shown.
+        self.problem_page.click_show()
+        self.assertTrue(self.problem_page.is_correct_choice_highlighted(correct_choices=[3]))
```

Comment:
Question, does the xpath in the https://github.com/edx/edx-platform/blob/fa705d83660e61e239040ba9e82b2645964caa9c/common/test/acceptance/pages/lms/problem.py#L412 check that there is exactly one checkmark, or just that it contains at least one of the expected class for the checkmark?